utils/src/main/java/com/revenuecat/purchases/utils/Responses.kt

```diff
@@ -1,7 +1,9 @@
 package com.revenuecat.purchases.utils
 
+import org.json.JSONObject
+
 object Responses {
```

Comment:
not related to your PR, but could this file be moved to test-utils?

Reply:
I am going to move it to another module we have called `test-utils`. It's not inside the `test` folder because stuff under test folder can't be shared across modules. `test-utils` has everything in a regular `src` folder and it's added to other modules via `testImplementation`.